includes/class-wc-payments.php

```diff
@@ -1356,6 +1352,19 @@ function ( $container ) {
 		}
 	}
 
+	/**
+	 * Initializes express checkout buttons if payments are enabled
+	 *
+	 * @return void
+	 */
+	public static function maybe_display_express_checkout_buttons() {
+		if ( WC_Payments_Features::are_payments_enabled() ) {
+			$payment_request_button_handler                = new WC_Payments_Payment_Request_Button_Handler( self::$account, self::get_gateway() );
+			$platform_checkout_button_handler              = new WC_Payments_Platform_Checkout_Button_Handler( self::$account, self::get_gateway(), self::$platform_checkout_util );
+			self::$express_checkout_button_display_handler = new WC_Payments_Express_Checkout_Button_Display_Handler( self::get_gateway(), $payment_request_button_handler, $platform_checkout_button_handler );
```

Comment:
```suggestion
			$express_checkout_button_display_handler = new WC_Payments_Express_Checkout_Button_Display_Handler( self::get_gateway(), $payment_request_button_handler, $platform_checkout_button_handler );
```
This is pretty pedantic, but since we moved the `is_woopay_enabled` functionality to the `WC_Payments_Features` class, does `$express_checkout_button_display_handler` need to be static anymore? Is it reused anywhere or can we just instantiate it here once as a local variable, like the other express checkout button handlers, and call it a day? 🤔

Reply:
Nice catch! It's only used here to instantiate the class now since we're no longer calling any methods on it. Fixed with [619bcca](https://github.com/Automattic/woocommerce-payments/pull/6187/commits/619bccad13ff41ba492fbd5bf444d4c6c2283017).